src/hotspot/share/opto/vectornode.cpp

```diff
@@ -2137,6 +2137,65 @@ bool MulVLNode::has_uint_inputs() const {
          has_vector_elements_fit_uint(in(2));
 }
 
+static Node* unsigned_min_max_xform(Node* n) {
```

Comment:
```suggestion
static Node* UMinMaxV_Ideal(Node* n) {
```
I think this is how we generally name functions when we share them between nodes of different types.